src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -652,6 +654,34 @@ public class SimpleEvents {
 					"\tcancel the event")
 				.since("2.7");
 		}
+
+		if (Skript.classExists("io.papermc.paper.event.player.PlayerStopUsingItemEvent")) {
+			Skript.registerEvent("Stop Using Item", SimpleEvent.class, PlayerStopUsingItemEvent.class, "[player] stop (using item|item use)")
+				.description("Called when a player stops using an item. For example, when the player releases the " +
+						"interact button when holding a bow, an edible item, or a spyglass.",
+						"Note that event-number/event-timespan will return the number of ticks the item was held for.")
+				.requiredPlugins("Paper 1.18.2+")
+				.examples(
+					"on player stop using item:",
+						"\tbroadcast \"%player% held %event-item% for %event-timespan%.\"")
+				.since("INSERT VERSION");
```

Comment:
```suggestion
					.description("Called when a player stops using an item. For example, when the player releases the " +
							"interact button when holding a bow, an edible item, or a spyglass.",
							"Note that event-number/event-timespan will return the number of ticks the item was held for.")
					.requiredPlugins("Paper 1.18.2+")
					.examples(
						"on player stop using item:",
							"\tbroadcast \"%player% held %event-item% for %event-timespan%.\"")
					.since("INSERT VERSION");
```